composer.json

```diff
@@ -20,7 +20,7 @@
         "magento/module-ui": "101.0.*||101.1.*",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "php": "^7.0",
+        "php": ">=7.2.0",
```

Comment:
Is this due to the braintree_php library?

Reply:
Correct!